src/services/pcn-synflood/src/base/StatsBase.cpp

```diff
@@ -25,10 +25,24 @@ void StatsBase::update(const StatsJsonObject &conf) {
 StatsJsonObject StatsBase::toJsonObject() {
```

Comment:
Hello @francescomessina, It is better to override this function in src/Stats.cpp. Generally speaking files under base should not be changed by the user.

Reply:
Hello @mauriciovasquezbernal, also in this particular case?  that we want to change the behavior of the container always

Reply:
Done. Thanks @mauriciovasquezbernal

Reply:
@mauriciovasquezbernal Is it ok now?

Reply:
Yes, it is.

Reply:
Can you approve? :-)